### ISSUE TITLE
Add support for Animated SVG's

### ISSUE THREAD

**Issue by ZakisM** [hecrj/iced#874]
Adding support for Animated SVG's would greatly improve the UX for many applications. There are tons of existing animated SVG's already made for use on the web (For [example](https://samherbert.net/svg-loaders/) loading spinners) so it would be great to be able to easly add these into Iced Apps.

**Comment by nwrkbiz** [comment]
Has this been implemented already?

**Comment by olebedev** [comment]
Yes, animation capability has been recently added in https://github.com/iced-rs/iced/pull/2757

**Comment by TobiasEggerDDES** [comment]
@olebedev 

Could you provide a snippet or some hints on how to use animated SVGs or GIFs in iced?
I am a bit lost. Particular since modifying and redrawing the SVG "manually" in `update` and `view` appears to be kinda slow.
